src/client/Client.cc

```diff
@@ -7445,9 +7445,6 @@ int Client::_do_setattr(Inode *in, struct ceph_statx *stx, int mask,
 			const UserPerm& perms, InodeRef *inp)
 {
   int issued = in->caps_issued();
-  union ceph_mds_request_args args;
-  bool kill_sguid = false;
-  int inode_drop = 0;
```

Comment:
It seems you have reverted the patches from https://github.com/ceph/ceph/pull/43286.